rest/admin_api_test.go

```diff
@@ -3635,11 +3635,13 @@ func TestNotExistentDBRequest(t *testing.T) {
 
 	// Request to non-existent db with valid credentials
 	resp := rt.SendAdminRequestWithAuth("PUT", "/dbx/_config", "", "random", "password")
-	assertStatus(t, resp, http.StatusForbidden)
+	assertStatus(t, resp, http.StatusUnauthorized)
+	require.Contains(t, resp.Body.String(), ErrInvalidLogin.Message)
```

Comment:
This is different than the status code assertion on `master`, which is still `Forbidden`

Reply:
> This looks slightly different than the code that went into `master` - is that intentional?

I wish it weren't different, but the code that handles inactiveDatabases makes the error codes different. As long as the error codes match for existing database but unable to access and non existent database, I think this is OK.

The starting state of 3.0 and 3.1 was always different https://docs.google.com/spreadsheets/d/1aXL9D7R54R0prYr-6ini-Qrk3Pj42k3gE4VQbP-lCaI/edit#gid=0 and I've preserved the error codes.